docs/index.md

```diff
@@ -207,6 +207,37 @@ The repository ID of the new reposityory.
 
 ---
 
+## key
+
+The "key" command manages keys (passwords) for accessing the repository.
+
+### Args
+
+* `subcommand`: Subcommand of the `key` command. Needs to be one of 'list', 'add', 'remove' or 'passwd'.
```

Comment:
Can we remove the trailing period (ditto for `key_id`)?

We were kind of inconsistent about this, so I pushed #102 to get rid of other tailing periods for args.

Reply:
Done.